examples/offloading-multiple-pods/setup.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+# shellcheck source=/dev/null
+source "$here/../common.sh"
+
+CLUSTER_NAME_ORIGIN=origin
+CLUSTER_NAME_DESTINATION1=destination1
+CLUSTER_NAME_DESTINATION2=destination2
+
+CLUSTER_LABLE_ORIGIN="topology.liqo.io/type=origin"
```

Comment:
```suggestion
CLUSTER_LABEL_ORIGIN="topology.liqo.io/type=origin"
```

The same for other occurrences, but I think that all labels can be dropped as unused